src/points/saga.test.ts

```diff
@@ -560,13 +563,20 @@ describe('watchAppMounted', () => {
     const mockGetPointsConfig = jest.fn()
     const mockAction = { type: AppActions.APP_MOUNTED }
 
+    jest.mocked(safely).mockImplementation((saga) => {
```

Comment:
instead of this, can we do `.provide([[call(getPointsConfig), {}], ...)` in the tests? that is the pattern that we've more commonly been using for mocking

Reply:
removed the hacky test for now. will follow up with an additional PR once figure out a proper solution.